README.rst

```diff
@@ -22,52 +22,55 @@ Getting resources
 +++++++++++++++++
 
 #. Create a working directory::
-    
-     mkdir ~/tx2_workdir
-     cd ~/tx2_workdir
+
+     export tx2_workdir=~/tx2_workdir
+     mkdir -p $tx2_workdir
+     cd $tx2_workdir
 #. Clone the kernel sources for the TX2 Deep Learning Kit from this repository::
 
-     git clone https://github.com/antmicro/tx2-deep-learning-kit-bsp.git ~/tx2_workdir/
-#. Download the `Linux for Tegra 28.2.1 for TX2 <https://developer.nvidia.com/embedded/dlc/tx2-driver-package-r2821>`_ driver package provided by NVIDIA for Jetson TX2 to ``tx2_workdir``.
+     git clone https://github.com/antmicro/tx2-deep-learning-kit-bsp.git
+#. Download the `Linux for Tegra 28.2.1 for TX2 <https://developer.nvidia.com/embedded/dlc/tx2-driver-package-r2821>`_ driver package provided by NVIDIA for Jetson TX2 to ``tx2_workdir``::
+
+     wget https://developer.nvidia.com/embedded/dlc/tx2-driver-package-r2821 -O Tegra186_Linux_R28.2.1_aarch64.tbz2
 #. Unpack this archive into the working directory::
 
-     sudo tar xpf Tegra186_Linux_R28.2.1_aarch64.tbz2
+     sudo tar xpof Tegra186_Linux_R28.2.1_aarch64.tbz2
 #. Download the `root filesystem <https://developer.nvidia.com/embedded/dlc/sample-root-filesystem-r2821>`_ to ``tx2_workdir``.
 #. Unpack the downloaded root filesystem into the ``Linux_for_Tegra`` directory::
 
-     sudo tar xpf Tegra_Linux_Sample-Root-Filesystem_R28.2.1_aarch64.tbz2 -C ./Linux_for_Tegra/rootfs
+     sudo tar xpof Tegra_Linux_Sample-Root-Filesystem_R28.2.1_aarch64.tbz2 -C Linux_for_Tegra/rootfs
```

Comment:
keeping the owner of rootfs files is really important. If you change the files owner to non-privileged user the system will not boot.

Reply:
also if you change the ownership of all the files in `/home` to `root` the regular users processes will not be able to access it's home folder